File: tourism/src/main/java/com/tourism/BookingItem.java
package com.tourism;

/**
 * Клас для збереження позиції бронювання
 */
public class BookingItem {
    private String roomType;
    private double price;

    public BookingItem(String roomType, double price) {
        this.roomType = roomType;
        this.price = price;
    }

    public String getRoomType() {
        return roomType;
    }

    public double getPrice() {
        return price;
    }
}

File: tourism-python/src/tourism/booking_item.py
class BookingItem:
    """
    Клас для збереження позиції бронювання
    """

    def __init__(self, room_type: str, price: float):
        self.room_type = room_type
        self.price = price

    def get_room_type(self):
        return self.room_type

    def get_price(self):
        return self.price
